Sky Atlas/Sky Atlas.html

```diff
@@ -18177,283 +18309,1069 @@ <h1>Sections & Primary Docs</h1>
+          <td>Core</td>
+          <td>The <code>Slope 1</code> parameter may be modified by the Core Executor Agents, in consultation with the Stability Scope Advisors. This process will be conducted through the Operational Weekly Governance Cycle or, if necessary, through out-of-schedule Executive Votes.</td>
+        </tr>
+        <tr>
+          <td>
+            <dfn>A.4.4 - SKY Staking Mechanism - SKY Staking - SKY-Backed Borrowing - SKY Borrow Rate - Rate Setting Mechanism - Rate Setting Parameters - Slope 2</dfn>
+          </td>
+          <td>SKY Staking Mechanism - SKY Staking - SKY-Backed Borrowing - SKY Borrow Rate - Rate Setting Mechanism - Rate Setting Parameters - Slope 2</td>
+          <td>Core</td>
+          <td><code>Slope 2</code> represents the spread of the SKY Borrow Rate at 100% Utilization over the SKY Borrow Rate at Target Utilization.</td>
+        </tr>
+        <tr>
+          <td>
+            <dfn>A.4.4 - SKY Staking Mechanism - SKY Staking - SKY-Backed Borrowing - SKY Borrow Rate - Rate Setting Mechanism - Rate Setting Parameters - Slope 2 - Slope 2 Current Value</dfn>
+          </td>
+          <td>SKY Staking Mechanism - SKY Staking - SKY-Backed Borrowing - SKY Borrow Rate - Rate Setting Mechanism - Rate Setting Parameters - Slope 2 - Slope 2 Current Value</td>
+          <td>Core</td>
+          <td>The current value of the <code>Slope 2</code> parameter is 12.575%.</td>
+        </tr>
+        <tr>
+          <td>
+            <dfn>A.4.4 - SKY Staking Mechanism - SKY Staking - SKY-Backed Borrowing - SKY Borrow Rate - Rate Setting Mechanism - Rate Setting Parameters - Slope 2 - Slope 2 Modification</dfn>
+          </td>
+          <td>SKY Staking Mechanism - SKY Staking - SKY-Backed Borrowing - SKY Borrow Rate - Rate Setting Mechanism - Rate Setting Parameters - Slope 2 - Slope 2 Modification</td>
+          <td>Core</td>
+          <td>The <code>Slope 2</code> parameter may be modified by the Core Executor Agents, in consultation with the Stability Scope Advisors. This process will be conducted through the Operational Weekly Governance Cycle or, if necessary, through out-of-schedule Executive Votes.</td>
+        </tr>
+        <tr>
+          <td>
+            <dfn>A.4.4 - SKY Staking Mechanism - SKY Staking - SKY-Backed Borrowing - SKY Borrow Rate - Rate Setting Mechanism - Rate Setting Parameters - Slope 2 - Slope 2 Methodology</dfn>
+          </td>
+          <td>SKY Staking Mechanism - SKY Staking - SKY-Backed Borrowing - SKY Borrow Rate - Rate Setting Mechanism - Rate Setting Parameters - Slope 2 - Slope 2 Methodology</td>
+          <td>Core</td>
+          <td><p>Under normal circumstances governance should set the <code>Slope 2</code> parameter based on the following formula:</p><p><code>Slope 2 = Slope 1 * alpha</code></p><p>The parameters of this formula are specified in the documents herein</p>.</td>
```

Comment:
The period at the end of this document is ending up on a new row. I think the solution is to move the period to be before </p>

```suggestion
          <td><p>Under normal circumstances governance should set the <code>Slope 2</code> parameter based on the following formula:</p><p><code>Slope 2 = Slope 1 * alpha</code></p><p>The parameters of this formula are specified in the documents herein.</p></td>
```